dnf/base.py

```diff
@@ -1231,6 +1235,14 @@ def _pattern_to_pkgname(pattern):
             else:
                 return (pattern,)
 
+        # compat: erase in 2.0.0
+        if isinstance(grp_id, dnf.comps.Environment) or \
+           isinstance(grp_id, dnf.comps.Group):
```

Comment:
IIRC it can be `isinstance(grp_id, (dnf.comps.Environment, dnf.comps.Group)). Just FYI...